src/Proxy.NET/Endpoints/Attendee.cs

```diff
@@ -0,0 +1,60 @@
+using System.Net;
+using System.Text.Json;
+using Proxy.NET.Models;
+using Proxy.NET.Services;
+
+namespace Proxy.NET.Endpoints;
+
+public static class Attendee
+{
+    private enum RequestType
+    {
+        AttendeeAdd,
+        AttendeeGetKey
+    }
+
+    public static void AttendeeEndpoints(this IEndpointRouteBuilder routes)
+    {
+        MapRoute.Post(routes, RequestType.AttendeeAdd, AttendeeAdd, Auth.Type.Jwt, "/attendee/event/{event_id}/register");
+        MapRoute.Get(routes, RequestType.AttendeeGetKey, AttendeeGetKey, Auth.Type.Jwt, "/attendee/event/{event_id}");
+    }
+
+    /// <summary>
+    /// Represents an asynchronous operation that can return a value.
+    /// </summary>
+    /// <returns>A task representing the asynchronous operation.</returns>
+    private static async Task AttendeeAdd(HttpContext context, RequestType requestType, string extPath)
```

Comment:
More semanticly, we should name this `AddAttendee`, as `ActionTarget` is more a common naming convention in .NET.

Reply:
fixed